site/guide/workflows/_workflow-step-types.qmd

```diff
@@ -79,46 +71,89 @@ Workflows cannot be saved until condition branches are connected to other steps.
 
 ::: {.callout title="Time stamp configuration is in the current user's timezone."}
 Other users viewing those fields or the workflow will see the value automatically in their timezone.
 
 :::
 
-To configure a wait step, select when you would like to [resume workflow:]{.smallcaps}
+To configure a wait step, select when you would like to [resume workflow]{.smallcaps}
 
 | Option | Required field(s) | Description |
 |---:|---|---|
 | After Time Interval  | [wait duration]{.smallcaps} | Wait for a set amount of time. Applies to all models under the workflow. |
 | At Specified Date | [wait until]{.smallcaps} | Wait until a specific timestamp. Applies to all models under the workflow. If the milestone date inputted is in the past, the next workflow step will display immediately. |
 | At Specified Date on Model Field | [model field]{.smallcaps}^[[Manage model inventory fields](/guide/model-inventory/manage-model-inventory-fields.qmd#inventory-field-types)] | Wait until a specific timestamp as defined by a model inventory field on a per model basis. If the selected field is empty^[Make model inventory fields **[required when creating models]{.smallcaps}** to ensure necessary fields have valid values.] or the milestone date inputted is in the past, the next workflow step will display immediately. |
-| Time Delta from Specified Date on Model Field | [model field]{.smallcaps},^[[Manage model inventory fields](/guide/model-inventory/manage-model-inventory-fields.qmd#inventory-field-types)] [time delta direction]{.smallcaps}, [wait duration]{.smallcaps} | Wait for a calculated time relative to a date in a model field on a per model basis. If the selected field is empty^[Make model inventory fields **[required when creating models]{.smallcaps}** to ensure necessary fields have valid values.] or the milestone date inputted is in the past, the next workflow step will display immediately. |
+| Time Delta from Specified Date on Model Field | [model field]{.smallcaps}, [time delta direction]{.smallcaps}, [wait duration]{.smallcaps} | Wait for a calculated time relative to a date in a model field on a per model basis. If the selected field is empty^[Make model inventory fields **[required when creating models]{.smallcaps}** to ensure necessary fields have valid values.] or the milestone date inputted is in the past, the next workflow step will display immediately. |
 : **{{< fa clock >}} Wait** step configuration {.hover tbl-colwidths="[30,15,45]"}
 
-### Workflow Logic
-
-#### {{< fa arrow-right-arrow-left >}} Run Workflow
+### {{< fa arrow-right-arrow-left >}} Run Workflow
 
 Triggers another workflow indicated by **[select workflow to run]{.smallcaps}**, allowing you to chain workflows.
 
-#### {{< fa circle-stop >}} End
+### {{< fa circle-nodes >}} Webhook
+
+When a workflow reaches a webhook step, the workflow will pause. Listen for a `POST` request to resumes the workflow:
+
+::: {.panel-tabset}
+
+#### Copy webhook URL
```

Comment:
The docs rendering of `.panel-tabset` are slightly confusing I think. It makes it look like the user has three distinct options to choose from instead of three required steps that all must be taken

Is there a rendering that puts the steps more in order?
 
1. copy the url
2. configure and copy the payload
3. copy the authn headers
4. put this all together and send the webhook request